packaging/dpkg/debian/control

```diff
@@ -21,8 +21,8 @@ Build-Depends: debhelper,
                openssl,
                php-all-dev,
                php-cli,
-               python3 (>= 3.12),
-               python3-dev (>= 3.12),
+               python3 (>= 3.12) <!no-python312>,
+               python3-dev (>= 3.12) <!no-python312>,
                python3-setuptools,
                python3-passlib
```

Comment:
We still keep passlib for the tests, we can probably remove setuptools.

Reply:
Debian packaging has a target to run the tests, those are run for example when we upload to the debian repositories.